src/secret/secret.tsx

```diff
@@ -0,0 +1,6 @@
+export function secretCheck(){
```

Comment:
Hinweis: leere .spec.tsx-Files (bzw. hier müsste es sich sowieso um .ts-Files handeln, da es nicht um eine React-Komponente geht) aber auch leere describe-Statements führen zu Fehlern in der Testausführung. So auch hier geschehen. Haben Sie die Tests ausgeführt bevor Sie den PR gestellt haben? Wenn ja muss Ihnen sowas auffallen.